### PR TITLE
DEM and SwimmingDEM, boost:: changed to Kratos::

### PR DESCRIPTION
**Pull request by maceligueta**

### CODE REVIEW

File: applications/swimming_DEM_application/custom_utilities/fields/fluid_field_utility.h [hunk 1]
```diff
@@ -30,7 +30,7 @@ void ImposeFieldOnNodes(ModelPart& r_model_part, const VariablesList& variables_
 
 virtual void ImposeVelocityOnNodes(ModelPart& r_model_part, const Variable<array_1d<double, 3> >& container_variable)
 {
-    boost::shared_ptr<VelocityField> p_vel_field = boost::static_pointer_cast<VelocityField>(mpVectorField);
+    Kratos::shared_ptr<VelocityField> p_vel_field = boost::static_pointer_cast<VelocityField>(mpVectorField);
```
Comment (maceligueta) [inline applications/swimming_DEM_application/custom_utilities/fields/fluid_field_utility.h]: @RiccardoRossi , we might have to do something else here, right? Boost is still there.

**Review by GuillermoCasas — DISMISSED** [review]
Nice.

**Comment by maceligueta** [comment]
Let's wait for @RiccardoRossi 's answer before merging.

**Comment by maceligueta** [comment]
@roigcarlo , any opinion about how to fix this?

**Comment by maceligueta** [comment]
I think static_pointer_cast should be added to define.h in the same fashion as make_shared, right? @RiccardoRossi @pooyan-dadvand @roigcarlo @jcotela

**Comment by roigcarlo** [comment]
Let me check it

**Comment by roigcarlo** [comment]
@maceligueta see #1374

**Comment by roigcarlo** [comment]
@maceligueta Now you should be able to use the Kratos:: version of any pointer cast.

**Review by roigcarlo — APPROVED** [review]

**Pull request merged**